main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_cloudtrail" "trail" {
+  name           = "trail${replace(title(var.project), " ", "")}${title(var.environment)}"
+  s3_bucket_name = "${var.s3_bucket_name}"
+
+  enable_logging                = true
+  include_global_service_events = true
+  is_multi_region_trail         = false
+  enable_log_file_validation    = false
```

Comment:
Perhaps these should be exposed as optional configuration variables? I wasn't sure how opinionated we wanted to be.

Reply:
Yeah, I think all of these make sense to be exposed with defaults that align with the resource.

I also think that we should surface:

- `s3_key_prefix` (and use it to build up the policy)
- `is_organization_trail`